src/core/server/metrics/analytics/ops_metrics_event_schema.ts

```diff
@@ -0,0 +1,298 @@
+          },
+        },
+      },
+      uptime_in_millis: {
+        type: 'long',
+        _meta: { description: 'the OS uptime' },
+      },
+      cpuacct: {
+        properties: {
+          control_group: {
+            type: 'text',
+            _meta: { description: "name of this process's cgroup" },
+          },
+          usage_nanos: {
+            type: 'date',
+            _meta: { description: "cpu time used by this process's cgroup" },
+          },
+        },
+        _meta: {
+          description: 'cpu accounting metrics, undefined when not running in a cgroup',
+          optional: true,
+        },
+      },
+      cpu: {
+        properties: {
+          control_group: {
+            type: 'text',
+            _meta: { description: "name of this process's cgroup" },
+          },
+          cfs_period_micros: {
+            type: 'long',
+            _meta: { description: 'the length of the cfs period' },
+          },
+          cfs_quota_micros: {
+            type: 'long',
+            _meta: { description: 'total available run-time within a cfs period' },
+          },
+          stat: {
+            properties: {
+              number_of_elapsed_periods: {
+                type: 'long',
+                _meta: { description: 'number of cfs periods that elapsed' },
+              },
+              number_of_times_throttled: {
+                type: 'long',
+                _meta: { description: 'number of times the cgroup has been throttled' },
+              },
+              time_throttled_nanos: {
+                type: 'long',
+                _meta: {
+                  description: 'total amount of time the cgroup has been throttled for',
+                },
+              },
+            },
+            _meta: {
+              description: 'current stats on the cfs periods',
+              optional: false,
+            },
+          },
+        },
+        _meta: {
+          description: 'cpu accounting metrics, undefined when not running in a cgroup',
+          optional: true,
+        },
+      },
+    },
+  },
+  response_times: {
+    properties: {
+      avg_in_millis: {
+        type: 'long',
+        _meta: { description: 'average response time' },
+      },
+      max_in_millis: {
+        type: 'long',
+        _meta: { description: 'maximum response time' },
+      },
+    },
+  },
+  requests: {
+    properties: {
+      disconnects: {
+        type: 'long',
+        _meta: { description: 'number of disconnected requests since startup' },
+      },
+      total: {
+        type: 'long',
+        _meta: { description: 'total number of requests handled since startup' },
+      },
+      statusCodes: {
+        // the responses can vary so we use a pass_through here
+        type: 'pass_through',
+        _meta: {
+          description:
+            'number of request handled per response status code as status_code:number pairs, for example: {"200": 3} ',
```

Comment:
@afharo how this for a hint on the shape of the `statusCodes` field?

Reply:
I love it! thanks!